cypress/commons/actions/generic/ScenarioParameters.js

```diff
@@ -70,8 +78,16 @@ function discard() {
   getParametersConfirmDiscardButton().click();
 }
 
+function discardAndContinue() {
+  getParametersDiscardAndContinueButton().click();
+}
+
+function cancelDiscardAndContinue() {
+  getParametersCancelDiscardAndContinueButton().click();
+}
+
 function launch(timeoutGetLaunchButton = 180) {
-  const alias = api.forgeAlias('reqRunScenarioAlias');
+  const alias = api.forgeAlias('reqSaveScenarioAlias');
```

Comment:
The previous alias name `reqRunScenarioAlias` seems more adapted for this `launch` function

Reply:
done